packages/server/billing/helpers/teamLimitsCheck.ts

```diff
@@ -25,7 +33,7 @@ const enableUsageStats = async (userIds: string[], orgId: string) => {
     .table('OrganizationUser')
     .getAll(r.args(userIds), {index: 'userId'})
     .filter({orgId})
-    .update({suggestedTier: 'pro'})
+    .update({suggestedTier: 'team'})
```

Comment:
Fixed tier here, that is probably the reason I was not able to see usage stats nagging snackbar